NEWS.md

```diff
@@ -1,3 +1,32 @@
+## 30.0.0 2020-05-15
+
+ * RMB-552 smart index recreation on module upgrade
+ * RMB-583 move f_unaccent and concat* from public to schema, fix search_path
+ * RMB-164 Incorrect moduleVersion and rmbVersion in rmb_internal table
+ * RMB-491 Migrate deprecated/removed Future calls in preparation for
+   vert.x4 migration
+ * RMB-575 Add "sqlExpressionQuery" parameter
+ * RMB-587 Do not use public schema, remove it from search_path
+ * RMB-588 investigate wrong value passed from "module_from" to ${version}
+ * RMB-594 Random and changing order of GenerateRunner class generation
+ * RMB-565 Review if all CQL to SQL generation code is using masking
+ * RMB-590 Persist schema.json in rmb_internal
+ * RMB-591 Remove or harmonize 'optimizedSql' execution path
+ * RMB-580 Unit tests in Jenkins fail which seem related to streamGet
+ * RMB-605 Combining characters (diacritics, umlauts) not found
+ * RMB-606 HridSettingsIncreaseMaxValueMigrationTest has stopped working
+ * RMB-607 ${myuniversity}_${mymodule}.gin_trgm_ops
+ * RMB-612 PostgresClientIT can not be executed in isolation
+ * RMB-246 migrate to reactive postgres client (vertx-pg-client)
+￼* RMB-600 PostgresClient.upsertBatch
+ * RMB-619 Update Vert.x to 3.9.0
+ * RMB-622 PgExceptionUtil.getMessage: Compose a message of all
+   PgException fields.
+ * RMB-569 Drop fromModuleVersion and tOps from table section, use
+   schema.json comparison instead
+ * RMB-610 Remove JobAPI code
+ * RMB-616 Update readme to specify permissionsRequired
```

Comment:
```suggestion
 * [RMB-552](https://issues.folio.org/browse/RMB-552) smart index recreation on module upgrade
 * [RMB-583](https://issues.folio.org/browse/RMB-583) move f_unaccent and concat* from public to schema, fix search_path
 * [RMB-164](https://issues.folio.org/browse/RMB-164) Incorrect moduleVersion and rmbVersion in rmb_internal table
 * [RMB-491](https://issues.folio.org/browse/RMB-491) Migrate deprecated/removed Future calls in preparation for
   vert.x4 migration
 * [RMB-575](https://issues.folio.org/browse/RMB-575) Add "sqlExpressionQuery" parameter
 * [RMB-587](https://issues.folio.org/browse/RMB-587) Do not use public schema, remove it from search_path
 * [RMB-588](https://issues.folio.org/browse/RMB-588) investigate wrong value passed from "module_from" to ${version}
 * [RMB-594](https://issues.folio.org/browse/RMB-594) Random and changing order of GenerateRunner class generation
 * [RMB-565](https://issues.folio.org/browse/RMB-565) Review if all CQL to SQL generation code is using masking
 * [RMB-590](https://issues.folio.org/browse/RMB-590) Persist schema.json in rmb_internal
 * [RMB-591](https://issues.folio.org/browse/RMB-591) Remove or harmonize 'optimizedSql' execution path
 * [RMB-580](https://issues.folio.org/browse/RMB-580) Unit tests in Jenkins fail which seem related to streamGet
 * [RMB-605](https://issues.folio.org/browse/RMB-605) Combining characters (diacritics, umlauts) not found
 * [RMB-606](https://issues.folio.org/browse/RMB-606) HridSettingsIncreaseMaxValueMigrationTest has stopped working
 * [RMB-607](https://issues.folio.org/browse/RMB-607) ${myuniversity}_${mymodule}.gin_trgm_ops
 * [RMB-612](https://issues.folio.org/browse/RMB-612) PostgresClientIT can not be executed in isolation
 * [RMB-246](https://issues.folio.org/browse/RMB-246) migrate to reactive postgres client (vertx-pg-client)
 * [RMB-600](https://issues.folio.org/browse/RMB-600) PostgresClient.upsertBatch
 * [RMB-619](https://issues.folio.org/browse/RMB-619) Update Vert.x to 3.9.0
 * [RMB-622](https://issues.folio.org/browse/RMB-622) PgExceptionUtil.getMessage: Compose a message of all
   PgException fields.
 * [RMB-569](https://issues.folio.org/browse/RMB-569) Drop fromModuleVersion and tOps from table section, use
   schema.json comparison instead
 * [RMB-610](https://issues.folio.org/browse/RMB-610) Remove JobAPI code
 * [RMB-616](https://issues.folio.org/browse/RMB-616) Update readme to specify permissionsRequired
```
Links added, spurious character removed.

Reply:
I'll add links in the GitHub changelog.

Reply:
The time to add clickable links to Jira issues is less than the additional time people need to manually enter the issue numbers when they want to find out what the title description actually means. RMB release notes are read by many people because many modules are based on RMB.

Reply:
OK.. Then for completenes I'll add links in complete NEWS file. (and not just for those for 30 release).

Reply:
Compromise: 
If you don't like clickable links in the NEWS.md you may add a link "release note with clickable links" after the date and version in the section header that is the only clickable link per version in NEWS.md.